### PR TITLE
Fix notes feed timeout reducing resultSize to 25.

### PR DESCRIPTION
**Pull request by pudiva**

### CODE REVIEW

**Comment by pudiva** [comment]
@spannerinworks @cdccollins  when you have the time, can you rereview this one, please?

File: lib/tasks/scheduler.rake [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace :hackney do
   end
 
   desc "Update notes in the graph DB"
-  task :update_graph_notes, [:enqueues, :wait_seconds] => [:environment] do |_t, args|
+  task :update_graph_notes, [:enqueues, :limit, :wait_seconds] => [:environment] do |_t, args|
```
Comment (spannerinworks) [inline lib/tasks/scheduler.rake:13]: where does limit go?

Comment (pudiva) [reply lib/tasks/scheduler.rake:13]: fixed

**Review by spannerinworks — APPROVED** [review]
👍

**Pull request merged**